modules/ROOT/pages/URL-customization.adoc

```diff
@@ -4,22 +4,18 @@ You can request to customize your domain to have an URL that uses a domain diffe
 
 [NOTE]
 ====
-* You must own the targeted domain or subdomain and delegate its management to Bonita.
-** Bonita manages the SSL certificate lifecycle (request, validation, and renewal).
-* This customization is a paid option.
+* You must own the targeted domain or subdomain and delegate its management to Bonitasoft.
+** Bonitasoft manages the SSL certificate lifecycle (request, validation, and renewal).
+* This customization is already included in the Bonita Cloud offer.
```

Comment:
Is it necessary to specify it ?
For the other functionalities we do not specify if it is included in the basic bonita cloud offer.